nni/runtime/platform/local.py

```diff
@@ -21,7 +21,8 @@
     os.makedirs(_outputdir)
 
 _nni_platform = trial_env_vars.NNI_PLATFORM
-if _nni_platform == 'local':
+_nni_trial_job_id = trial_env_vars.NNI_TRIAL_JOB_ID
+if _nni_platform == 'local' and _nni_trial_job_id != 'runner':
```

Comment:
Where is trial job ID set to `runner`? I can't find the code.

Reply:
in `trial_runner.py`